src/gluonts/distribution/lds.py

```diff
@@ -473,40 +485,38 @@ class LDSArgsProj(mx.gluon.HybridBlock):
     def __init__(
         self,
         output_dim: int,
-        noise_std_ub: float = 1.0,
-        innovation_ub: float = 0.01,
+        noise_std_bounds: ParameterBounds,
+        innovation_bounds: ParameterBounds,
     ) -> None:
         super().__init__()
         self.output_dim = output_dim
         self.dense_noise_std = mx.gluon.nn.Dense(
-            units=1,
-            flatten=False,
-            activation="softrelu"
-            if noise_std_ub is float("inf")
-            else "sigmoid",
+            units=1, flatten=False, activation="sigmoid"
         )
         self.dense_innovation = mx.gluon.nn.Dense(
-            units=1,
-            flatten=False,
-            activation="softrelu"
-            if innovation_ub is float("inf")
-            else "sigmoid",
+            units=1, flatten=False, activation="sigmoid"
         )
         self.dense_residual = mx.gluon.nn.Dense(
             units=output_dim, flatten=False
         )
 
-        self.innovation_factor = (
-            1.0 if innovation_ub is float("inf") else innovation_ub
-        )
-        self.noise_factor = (
-            1.0 if noise_std_ub is float("inf") else noise_std_ub
-        )
+        self.innovation = innovation_bounds
+        self.noise_std = noise_std_bounds
```

Comment:
minor: change the names of the member variables as well to include "bounds" in the name.

Reply:
Thanks! I missed that!